upgrading/index.adoc

```diff
@@ -0,0 +1,104 @@
+[[install-config-upgrading-index]]
+= Upgrade Methods and Strategies
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[upgrading-introduction]]
+== Introduction to Upgrading Clusters
+
+When new versions of {product-title} are released, you can upgrade your existing
+cluster to apply the latest enhancements and bug fixes.
+ifdef::openshift-origin[]
+For OpenShift Origin, see the
+https://github.com/openshift/origin/releases[Releases page] on GitHub to review
+the latest changes.
+endif::[]
+ifdef::openshift-enterprise[]
+This includes upgrading from previous minor versions, such as release 3.6 to
+3.7, and applying asynchronous errata updates within a minor version (3.7.z
+releases). See the xref:../release_notes/ocp_3_7_release_notes.adoc#release-notes-ocp-3-7-release-notes[{product-title} 3.7 Release Notes] to review the latest changes.
+
+[NOTE]
+====
+Due to the xref:../release_notes/v2_vs_v3.adoc#release-notes-v2-vs-v3[core architectural changes]
+between the major versions, OpenShift Enterprise 2 environments cannot be
+upgraded to {product-title} 3 and require a fresh installation.
+====
+endif::[]
+
+Unless noted otherwise, node and masters within a major version are forward and
+backward compatible
+xref:../install_config/upgrading/automated_upgrades.adoc#preparing-for-an-automated-upgrade[across
+one minor version], so upgrading your cluster should go smoothly. However, you
+should not run mismatched versions longer than necessary to upgrade the entire
+cluster.
+
+[[install-config-upgrading-methods]]
+== Upgrade Methods
+
+There are two methods available for performing {product-title} cluster upgrades:
+automated or manual.
+
+[[install-config-upgrading-methods-automated]]
+=== Automated Method
+
+The automated upgrade method uses Ansible playbooks to automate the tasks needed
+to upgrade a {product-title} cluster. You can use the inventory file that you
+used during inital installation to run the upgrade playbooks. Using this method
+allows you to choose between either
+xref:install-config-upgrading-strategies[upgrade strategy], in-place upgrades or
```

Comment:
[upgrade strategy]: in-place upgrades or blue green deployments.

(The comma tripped me up at first, as I thought it was a list.)

Reply:
Fixt